docs/docs/user/getting-started/setup.md

```diff
@@ -78,7 +78,8 @@ To deploy Kafka, this guide uses Confluent's [Helm chart](https://github.com/con
         docker push localhost:12345/kafka-connect-jdbc:7.1.3
         ```
 
-    Detailed instructions on building, tagging and pushing a docker image can be found in [Docker docs](https://docs.docker.com/).
+    Detailed instructions on building, 
+    tagging and pushing a docker image can be found in [Docker docs](https://docs.docker.com/).
```

Comment:
why the linebreak?

Reply:
The line was longer than 120 characters. In general, we should keep the lines short. This makes it way easier to read the diffs